spec/stubs/payouts.rb

```diff
@@ -0,0 +1,100 @@
+module Stubs
+  class Payout
+    class << self
+      def success_get(payout_id)
+        response_body = {
+          "id": 1,
+          "type": "payout",
+          "data": {
+            "id": "po_xyz",
+            "account_id": "449e7a5c-69d3-4b8a-aaaf-5c9b713ebc65",
```

Comment:
Are we surfacing the wrong id here? This should be a public id

Reply:
Oh we are, but only in our docs. Just tested the API and we're surfacing the public id.
I fixed it here and we can change our docs in a follow-up